test_create.py

```diff
@@ -0,0 +1,39 @@
+from commands import CreateCommand   # The code to test
```

Comment:
move this to tests folder

Reply:
Actually, it raises an error when I import a file from another parent directory. I have searched a lot for a solution and nothing seems to work. so if you know a good way please share it.

Reply:
try something like this.
```
# Put the repo root dir in PATH so script can find the dblib folder
repo_root_dir = os.path.join(os.path.abspath(os.path.dirname(__file__)), os.pardir)
sys.path.append(repo_root_dir)
from dblib.Database import ArgusClient
```
This will help